src/Middleware.php

```diff
@@ -126,7 +126,9 @@ protected function get(string $key): ?FulfilledPromise
     protected function save(string $key, ?ResponseInterface $response = null, int $ttl): bool
     {
         if ($response && $response->getStatusCode() === 200) {
-            return $this->cache->set($key, (string)$response->getBody(), $ttl) ?? true;
+            $this->cache->set($key, (string)$response->getBody(), $ttl);
+            $response->getBody()->rewind();
+            return true;
```

Comment:
Great! Maybe just return the actual bool of cache->set()?

Reply:
i do this, because $this->cache->set return void or null but not boolean

Reply:
According to the [PSR-16 interface, it should return a boolean](https://github.com/php-fig/simple-cache/blob/master/src/CacheInterface.php#L29). So maybe just do:
```
$saved = $this->cache->set($key, (string)$response->getBody(), $ttl) ?? true;
$response->getBody()->rewind();
return $saved;
```
(Even if currently, the return value is not used - it may be at some point in the future. ;)

Reply:
ok, this solution I liked too

Reply:
so, what about merge request??

Reply:
Can't you correct it so as to return the value?

Reply:
fixed

Reply:
Hi, I fix own code. Is it good?